eventsourcing/src/test/scala/com/evolutiongaming/akkaeffect/eventsourcing/JournalKeeperTest.scala

```diff
@@ -466,13 +487,13 @@ object JournalKeeperTest {
   def journalKeeperOf[F[_]: Async](
     seqNr: SeqNr,
     state: F[Unit],
-    snapshotOffer: Option[SnapshotMetadata],
+    metadata: Option[SnapshotMetadata],
```

Comment:
IMHO `snapshotMetadata` makes more sense

Reply:
I do not agree